prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -468,25 +502,12 @@ ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_pee
 	dlist_insert_tail(&tx_entry->tx_entry_entry, &ep->tx_entry_list);
 #endif
 
-	err = rxr_pkt_post_ctrl(ep, RXR_TX_ENTRY, tx_entry, RXR_EAGER_RTW_PKT, 0);
+	err = rxr_pkt_post_ctrl(ep, RXR_TX_ENTRY,
+				tx_entry, RXR_EAGER_RTW_PKT, 0);
 
 	if (OFI_UNLIKELY(err))
 		return err;
 
-	current = ofi_gettime_us();
```

Comment:
this section was removed?

Reply:
Yeah. There's timing logic in `rxr_pkt_wait_handshake` because we do not want to wait for handshake packet forever. There is no such logic in `rxr_pkt_trigger_handshake`

Reply:
In this commit ```rxr_pkt_wait_handshake``` does not have the timing logic either. It got lost

Reply:
Ok. Thanks for reminding! I will add it.

Reply:
Timing logic added for `rxr_pkt_wait_handshake`, resolving.